x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -476,47 +475,16 @@ export class ExplorerUI extends React.Component {
                   </EuiFlexItem>
                 </EuiFlexGroup>
 
-                <EuiFlexGroup
-                  direction="row"
-                  gutterSize="l"
-                  responsive={true}
-                  className="ml-anomalies-controls"
-                >
-                  <EuiFlexItem grow={false} style={{ width: '170px' }}>
-                    <EuiFormRow
-                      label={i18n.translate('xpack.ml.explorer.severityThresholdLabel', {
-                        defaultMessage: 'Severity threshold',
-                      })}
-                    >
-                      <SelectSeverity />
-                    </EuiFormRow>
+                <EuiFlexGroup direction="row" gutterSize="l" responsive={true} alignItems="center">
+                  <EuiFlexItem grow={false}>
+                    <SelectSeverity c />
```

Comment:
If the table severity threshold means than no anomalies are displayed, I'm seeing a `0` appear above the panel:

![image](https://user-images.githubusercontent.com/7405507/119812554-2a814a00-bee0-11eb-9839-c7c4ab45f6f1.png)

Reply:
Actually, it was because of the AnomaliesMap wrapper (not part of this PR). Fixed in https://github.com/elastic/kibana/pull/99489/commits/c3c90ac232d868532e28854a342fa926acb6a76c